README.md

```diff
@@ -129,6 +129,92 @@ The image ships with some basic fonts (`font-terminus font-inconsolata font-deja
 - Alpine Linux see https://wiki.alpinelinux.org/wiki/Fonts
 - Debian/Ubuntu see https://wiki.debian.org/Fonts
 
+#### Custom fonts directory
+
+Let's say your building a docker image for you application, where you want to use the `ffmpeg` and `ffprobe` binaries, and  want to have a dedicated directory for storing fonts, e.g. `/app/fonts`. In order to achieve this, let's take a look at the following steps.
+
+1. The library `fontconfig` has to be installed in your image and the cache directory has to be writable (for non-root users).
+
+```Dockerfile
+RUN <<EOT bash
+  set -ex
+  apt-get update
+  apt install -y fontconfig
+  apt-get clean
+  rm -rf /var/lib/apt/lists/*
+  chmod -R 777 /var/cache/fontconfig
+EOT
+```
+
+2. A custom config file containing the fonts directory you want use has to be added to the image. Here, a file `50-custom.conf` will be added to the `/etc/fonts/conf.d` directory, containing an additional `/app/fonts` directory for Fontconfig to use.
+
+```Dockerfile
+COPY <<EOF /etc/fonts/conf.d/50-custom.conf
+<?xml version="1.0"?>
+<!DOCTYPE fontconfig SYSTEM "urn:fontconfig:fonts.dtd">
+<fontconfig>
+        <dir>/app/fonts</dir>
+</fontconfig>
+EOF
+```
+
+3. The `/etc/fonts` directory must *not* be copied to the image to prevent installation conflicts with the already installed `fontconfig` (from previous step). Only copy the binaries.
+
+```Dockerfile
+COPY --from=mwader/static-ffmpeg:7.0.1 /ffmpeg /usr/bin/
+COPY --from=mwader/static-ffmpeg:7.0.1 /ffprobe /usr/bin/
+```
+
+4. Make sure the `/app/fonts` directory exist in your image.
+
+```Dockerfile
+WORKDIR /app
+
+RUN <<EOT bash
+  set -ex
+  mkdir -p ./{fonts}
+EOT
+```
+
+If you followed the steps above, your image should look something like this.
+
+```Dockerfile
+FROM python:3.11-slim-bookworm AS base
+
+RUN <<EOT bash
+  set -ex
+  apt-get update
+  apt install -y fontconfig
+  apt-get clean
+  rm -rf /var/lib/apt/lists/*
+  chmod -R 777 /var/cache/fontconfig
+EOT
+
+COPY <<EOF /etc/fonts/conf.d/50-custom.conf
+<?xml version="1.0"?>
+<!DOCTYPE fontconfig SYSTEM "urn:fontconfig:fonts.dtd">
+<fontconfig>
+        <dir>/app/fonts</dir>
+</fontconfig>
+EOF
+
+FROM base as ffmpeg
+
+COPY --from=mwader/static-ffmpeg:7.0.1 /ffmpeg /usr/bin/
+COPY --from=mwader/static-ffmpeg:7.0.1 /ffprobe /usr/bin/
+
+FROM ffmpeg AS app
+
+WORKDIR /app
+
+RUN <<EOT bash
+  set -ex
+  mkdir -p ./{fonts}
```

Comment:
Some tick going on using `./{...}`? 🤔

Reply:
I'm not exactly sure what you mean with "tick", but normally I do this if I want to create multiple directories with single mkdir statement (a bit silly with only 1 directory now I agree).

But now I realsed that if you do this with one directory, e.g. `mkdir -p ./{only-one-directory}` it will literally create the directory as `./{only-one-directory}` instead of `./only-one-directory`.

Think `mkdir -p ./fonts` or `mkdir -p ./{fonts,}` should then be fine.

Reply:
> I'm not exactly sure what you mean with "tick", but normally I do this if I want to create multiple directories with single mkdir statement (a bit silly with only 1 directory now I agree).

I suspect is was something like that

> But now I realsed that if you do this with one directory, e.g. `mkdir -p ./{only-one-directory}` it will literally create the directory as `./{only-one-directory}` instead of `./only-one-directory`.
> 
> Think `mkdir -p ./fonts` or `mkdir -p ./{fonts,}` should then be fine.

Ah oh TIL :)